app/src/router/AppRouter.tsx

```diff
@@ -62,6 +64,10 @@ export const routeMap: Routes = {
         page: <UnauthorisedPage />,
         type: ROUTE_TYPE.PUBLIC,
     },
+    [FEEDBACK]: {
```

Comment:
Private route as discussed in tech huddle

Reply:
Change to a private route. 
Also, amend implementation for the phase banner to only display the word "feedback" as a link when logged in